draft-packages/form/docs/TextField.stories.tsx

```diff
@@ -200,7 +200,7 @@ const StickerSheetTemplate: StoryFn<{ isReversed: boolean }> = ({
 export const StickerSheetDefault = StickerSheetTemplate.bind({})
 StickerSheetDefault.storyName = "Sticker Sheet (Default)"
 StickerSheetDefault.parameters = {
-  chromatic: { disable: false },
+  chromatic: { disable: false, delay: 400, pauseAnimationAtEnd: true },
```

Comment:
Will these work without the `delay`?

Reply:
I'm confused, didn't you suggested to add it to see if chromatic would wait for the ticks to show up...? 😅